test/gui/shared/steps/spaces_context.py

```diff
@@ -46,28 +45,19 @@ def step(context, user, space_name):
     waitForInitialSyncToComplete(getResourcePath('/', user, space_name))
 
 
-@Then('user "|any|" should be able to open the file "|any|" on the file system')
-def step(context, user, file_name):
-    file_path = getResourcePath(file_name, user)
-    test.compare(can_read(file_path), True, "File should be readable")
-
-
-@Then('as "|any|" the file "|any|" should have content "|any|" on the file system')
-def step(context, user, file_name, content):
-    file_path = getResourcePath(file_name, user)
-    file_content = read_file_content(file_path)
-    test.compare(file_content, content, "Comparing file content")
-
-
-@Then('user "|any|" should not be able to edit the file "|any|" on the file system')
-def step(context, user, file_name):
-    file_path = getResourcePath(file_name, user)
-    test.compare(not can_write(file_path), True, "File should not be writable")
-
-
 @Then(
     'as "|any|" the file "|any|" of space "|any|" should have content "|any|" in the server'
 )
 def step(context, user, file_name, space_name, content):
     downloaded_content = get_file_content(space_name, file_name, user)
     test.compare(downloaded_content, content, "Comparing file content")
+
+
+@Then(
+    r'as "([^"]*)" the space "([^"]*)" should have (?:folder|file) "([^"]*)" in the server',
+    regexp=True,
+)
+def step(context, user, space_name, resource_name):
+    print(user, space_name, resource_name)
+    exists = resource_exists(space_name, resource_name, user)
+    test.compare(exists, True, "Resource should exist")
```

Comment:
message will be added to the test result no matter whether the test passed or failed.
https://doc.qt.io/squish/squish-api.html#boolean-test-compare-value1-value2-message please visit this link.
Don't you think some useful message should be given which help us to understand comparsion

Reply:
Yeah, fully aware. 
Won't it be enough? Suggest me some

![Screenshot from 2023-02-03 17-13-25](https://user-images.githubusercontent.com/52366632/216592991-fedd22f0-afb2-44e9-ae71-4899ecf63d96.png)

![Screenshot from 2023-02-03 17-14-52](https://user-images.githubusercontent.com/52366632/216592847-4a2409a4-3e36-4874-9271-ab03ed33a467.png)

Reply:
A message which tells what this function does like 
`test.compare(downloaded_content, content, "Comparing file content")`
Something like this but not exactly
```suggestion
    test.compare(exists, True, "In server looking for 'xyz' resource in space 'abc' ")
```

Reply:
The step says that. So, I think I will leave it as it is.

Reply:
But I don't like `Resource should exist` message it confusing

Reply:
Rephrased to `Resource exists`